src/debugtopus.ts

```diff
@@ -89,6 +89,18 @@ export const prepareTestRun = async ({
   return { testFilePath, configFilePath, outputDir, packageRootDir };
 };
 
+export const createPlaywrightCommand = ({
+  configFilePath,
+  testFilePath,
+}: {
+  configFilePath: string;
+  testFilePath: string;
+}): string =>
+  `npx playwright test --config=${configFilePath.replaceAll(
```

Comment:
are we sure this is the right fix? https://github.com/microsoft/playwright/issues/19563 looks like double backslashes is the intended way?

Reply:
I tested it out and it works... it was quite a journey yesterday... you can try it yourself if you have a windows machine...

Last comment [here](https://stackoverflow.com/questions/71053528/playwright-no-tests-found-on-simple-copy-of-example-spec-ts)

Reply:
ok, i figured you tested it, just seems a bit weird, but lgtm